docs/reference/api/docker_remote_api_v1.18.md

```diff
@@ -16,6 +16,7 @@ weight = 3
  - The Remote API has replaced `rcli`.
  - The daemon listens on `unix:///var/run/docker.sock` but you can
    [Bind Docker to another host/port or a Unix socket](../commandline/dockerd.md#bind-docker-to-another-host-port-or-a-unix-socket).
+[//]: # (the primary purpose of the link is to generate our online docs (and not to be read on GitHub). it cannot be linked to the correct content on GitHub.)
```

Comment:
hm, I don't think we need the note, because the reviewers are aware of this, but if we *do* add a note possibly make it a TODO (e.g. `TODO this anchor is broken on GitHub, but works on docs.docker.com. Update this once https://github.com/docker/docs-base/issues/186 is resolved`